packages/app-content-pages/src/screens/Team/Team.js

```diff
@@ -0,0 +1,84 @@
+import counterpart from 'counterpart'
+import { Box, Button, Heading } from 'grommet'
+import { array, arrayOf, bool, func, shape, string } from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import Team from './components/Team'
+import en from './locales/en'
+import Head from '../../shared/components/Head'
+import TwoColumnLayout from '../../shared/components/TwoColumnLayout'
+
+counterpart.registerTranslations('en', en)
+
+const StyledButton = styled(Button)`
+  ${props => props.active && `
+    background: none;
+    font-weight: bold;
+  `}
+`
+
+function TeamComponent (props) {
+  const { className, data, filters } = props
+
+  const main = (
+    <article>
+      <Heading margin={{ top: 'none' }} size='small'>
+        Our Team
+      </Heading>
+
+      {data && data.map(team => (
+        <Team
+          key={team.id}
+          name={team.name}
+          people={team.people}
+        />
+      ))}
+    </article>
+  )
+
+  const sidebar = (
+    <Box gap='small'>
+      {filters.map(filter => (
+        <div key={filter.name} >
```

Comment:
Not sure what's best for accessibility, but this sidebar is a menu, so I was thinking initially that this should be an unordered list, but I've read that sometimes lists can be bad for accessibility. @eatyourgreens do you have a suggestion?

Reply:
Lists of links should be marked up as lists.